### PR TITLE
created ASUser

### PR DESCRIPTION
**Pull request by ghost**
Comprobar que ASUser funciona

### CODE REVIEW

File: HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+			User user = null;
+			try {
+				user =  (User) query.getSingleResult();
+			}
+			catch (NoResultException ex) {
+				System.out.println(ex.getMessage());
+			}
+			
+			if (user != null){
+				tr.rollback();
```
Comment (GasanNazer) [inline HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java:37]: You do not need this if else. You can write just if(user == null) //create the user and result = true;. And previously you need to define your boolean result = false;

File: HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+				em.close();
+				emf.close();
+				return false;
+			}
+			
+			else {
+				user = new User (tUser.getNickname(),
+						tUser.getRating(),
+						tUser.getDescription(),
+						false,
```
Comment (GasanNazer) [inline HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java:47]: The two variables that indicates whether you have a host or a traveler or both should be taken from your transfer just as the nickname,rating and description.

File: HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java (hunk 1 above)
Comment (SirTorresDev) [reply HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java:37]: Estoy de acuerdo

**Review by SirTorresDev — COMMENTED** [review]
It's ok, buy you can modify the transfer parameters

**Pull request merged**